com.unity.shadergraph/Editor/GraphUI/EditorCommon/Preview/PreviewManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using Editor.GraphUI.Utilities;
+using UnityEditor.ShaderGraph.GraphDelta;
+using UnityEngine;
+using UnityEditor.ShaderGraph.GraphUI.DataModel;
+
+namespace UnityEditor.ShaderGraph.GraphUI.EditorCommon.Preview
+{
+
+using PreviewRenderMode = HeadlessPreviewManager.PreviewRenderMode;
+
+	public class PreviewManager
```

Comment:
not sure why the formatting is so atrocious on github, will fix this before merging